{
  "Header": {
    "Original": "CVE-2018-16884 (A flaw was found in the Linux kernel's NFS41+ subsystem. NFS41+ shares ...)",
    "Line": 331499,
    "ID": "CVE-2018-16884",
    "Description": "(A flaw was found in the Linux kernel's NFS41+ subsystem. NFS41+ shares ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-1771-1 DLA-1731-1}",
      "Line": 331500,
      "Type": "xref",
      "Bugs": [
        "DLA-1771-1",
        "DLA-1731-1"
      ]
    },
    {
      "Original": "- linux 4.19.16-1",
      "Line": 331501,
      "Type": "package",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "4.19.16-1"
    },
    {
      "Original": "[stretch] - linux 4.9.161-1",
      "Line": 331502,
      "Type": "package",
      "Release": "stretch",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "4.9.161-1"
    },
    {
      "Original": "NOTE: https://patchwork.kernel.org/cover/10733767/",
      "Line": 331503,
      "Type": "NOTE",
      "Description": "https://patchwork.kernel.org/cover/10733767/"
    },
    {
      "Original": "NOTE: https://patchwork.kernel.org/patch/10733769/",
      "Line": 331504,
      "Type": "NOTE",
      "Description": "https://patchwork.kernel.org/patch/10733769/"
    },
    {
      "Original": "NOTE: https://bugzilla.redhat.com/show_bug.cgi?id=1660375",
      "Line": 331505,
      "Type": "NOTE",
      "Description": "https://bugzilla.redhat.com/show_bug.cgi?id=1660375"
    }
  ]
}